pkg/server/admin_test.go

```diff
@@ -1060,6 +1059,10 @@ func TestClusterAPI(t *testing.T) {
 			// We need to retry, because the cluster ID isn't set until after
 			// bootstrapping.
 			testutils.SucceedsSoon(t, func() error {
+				// In particular, we need to repeatedly set this value, as there is
```

Comment:
Would it be possible to detect when the SQL migration is complete and just wait for that right after starting the server?

Reply:
I've switched to using `SET CLUSTER SETTING` which doesn't have this problem in the first place. Thanks for giving me that *brilliant* idea 😄